corehq/apps/commtrack/models.py

```diff
@@ -193,6 +157,7 @@ class CommtrackConfig(QuickCachedDocumentMixin, Document):
     multiaction_enabled = BooleanProperty()
     multiaction_keyword_ = StringProperty()
 
+    # todo: remove?
     requisition_config = SchemaProperty(CommtrackRequisitionConfig)
     openlmis_config = SchemaProperty(OpenLMISConfig)
```

Comment:
Ha! I removed a lot of stuff in https://github.com/dimagi/commcare-hq/pull/12414 but I guess there were other pockets not revealed by the things I was grepping for / the thread I was pulling on. I'll add this comment thread to that ticket